core/engine/src/value/mod.rs

```diff
@@ -354,6 +354,13 @@ impl JsValue {
         self.0.is_integer32() || self.0.is_float64()
     }
 
+    /// Returns true if the value is a negative zero (`-0`).
+    #[inline]
+    #[must_use]
+    pub fn is_negative_zero(&self) -> bool {
```

Comment:
issue: does this need to be public?

I'm not sure that I'm convinced we should support this method in the public API ...